content/minfraud/sandbox-environment.mdx

```diff
@@ -0,0 +1,79 @@
+---
+draft: false
+title: Testing minFraud in the MaxMind Sandbox
+---
+
+MaxMind maintains a Sandbox environment so that you can do basic integration
+tests with the minFraud services. In order to set up a Sandbox account you must
+already have a paid account, and you must be an account administrator. [Learn
+how to set up a Sandbox account on our Knowledge
+Base.](https://support.maxmind.com/hc/en-us/articles/8604168846363)
```

Comment:
There seems to be a lot of duplicate content for the minFraud and GeoIP sandbox files. It might make sense to condense this and use a shared template for the similar parts. This would save others time in the future to only have to update one file rather than updating two with the same exact content.